src/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -32,6 +32,8 @@ public enum EncryptionPolicy
         RequireEncryption = 0,
     }
     public delegate System.Security.Cryptography.X509Certificates.X509Certificate LocalCertificateSelectionCallback(object sender, string targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection localCertificates, System.Security.Cryptography.X509Certificates.X509Certificate remoteCertificate, string[] acceptableIssuers);
+    public delegate X509Certificate ServerCertificateSelectionCallback(object sender, string hostName);
```

Comment:
Let's treat this file as informal API design discussion and other files as reference

Reply:
ref definitions need fully qualified type name: "System.Security.Cryptography.X509Certificates.X509Certificate" like the delegate definition above.